### PR TITLE
Update: Broadcasted countdown's to server, new menu look

### PR DESCRIPTION
**Pull request by zunnu**

### CODE REVIEW

**Comment by dgibbs64** [comment]
Any update on this?

**Comment by zunnu** [comment]
No sorry not yet I have being busy with other stuff I will continue this soon.

**Comment by dgibbs64** [comment]
No problem. I think that this feature will be a popular one :)

File: lgsm/functions/core_getopt.sh [hunk 1]
```diff
@@ -44,28 +53,67 @@ case "$getopt" in
...
+	red=`echo -en "\e[31m"`
+ 	green=`echo -en "\e[32m"`
+ 	blue=`echo -en "\e[34m"`
+ 	lightred=`echo -en "\e[91m"`
+ 	lightgreen=`echo -en "\e[92m"`
+ 	cyan=`echo -en "\e[96m"`
+ 	purple=`echo -en "\e[95m"`
+
+
+		echo ""
```
Comment (dgibbs64) [inline lgsm/functions/core_getopt.sh]: this needs removing

**Pull request closed without merging**